pkg/provider/kubernetes/crd/traefikio/v1alpha1/middleware.go

```diff
@@ -49,6 +49,7 @@ type MiddlewareSpec struct {
 	Retry             *Retry                     `json:"retry,omitempty"`
 	ContentType       *dynamic.ContentType       `json:"contentType,omitempty"`
 	GrpcWeb           *dynamic.GrpcWeb           `json:"grpcWeb,omitempty"`
+	Coraza            *Coraza                    `json:"coraza,omitempty"`
```

Comment:
I'd call it CorazaWAF or WAF. Coraza by itself is not too intuitive.

Reply:
First I was thinking to make coraza under waf category, but it might be that there are not many waf implementations really? Perhaps the best is to use `CorazaWAF`, because it might be a problem to use just `WAF` if someone wants to add second waf middleware in future